common/tests/test_swaglog.cc

```diff
@@ -39,26 +39,25 @@ void recv_log(int thread_cnt, int thread_msg_cnt) {
 
     REQUIRE(buf[0] == CLOUDLOG_DEBUG);
```

Comment:
This test fail, but I'm not sure why it's here in the first place. If you got a JSON as a string, then buf[0] should be '{' right ?

Reply:
see https://github.com/commaai/openpilot/blob/78e20cff3906ee76c356e8bdef920397e60389b4/common/swaglog.cc#L94